kettle/Makefile

```diff
@@ -38,6 +38,9 @@ push: build
 deploy: get-cluster-credentials
 	sed "s/:latest/:$(TAG)/g" deployment.yaml | kubectl apply -f -
 
+deploy-monitor: get-cluster-credentials
```

Comment:
Can we please have bazel rules to build, push and deploy this like we do for testgrid and prow?

Reply:
rules_docker doesn't support a `RUN` equivalent, so it would be pretty hard to convert kettle/Dockerfile to install gcloud and python requirements. Feel free to file a bug.

Reply:
There are already base images that have gcloud installed.

There are already rules for installing pip dependencies -- https://github.com/bazelbuild/rules_python/blob/master/docs/python/pip.md#pip_import